app/assets/stylesheets/utilities/_colors.scss

```
@@ -6,6 +6,18 @@
   color: $text-color-lightest;
 }
 
+.icon-white {
+  color: $white;
+}
+
+.icon-discord {
+  color: $discord;
+}
+
+.icon-battlenet {
+  color: $bnet;
+}
```

Let's use text-* instead to be more consistent with the others, I think `text-discord` and `text-white` already exists even

```suggestion
.text-battlenet {
  color: $bnet;
}
```

---

I was going to but I figured maybe if there was a case where you wanted button specific CSS but it is pretty unlikely so I'll change them to use the text ones.